src/functions/src/test/java/org/apache/jmeter/functions/TestTimeShiftFunction.java

```diff
@@ -106,13 +109,35 @@ public void testNowPlusOneDay() throws Exception {
         assertThat(tomorrowFromFunction, sameDay(tomorrow));
     }
 
+    private int dstOffset(Date from, Date to) {
```

Comment:
Why is this required?

Reply:
Because it is called from line 139?
But seriously that function is the essence of this PR, as it calculates the DST offset between two dates for the timezone of the JVM.